Tests/DiligentCoreAPITest/src/TestingEnvironment.cpp

```diff
@@ -402,6 +403,63 @@ RefCntAutoPtr<ITexture> TestingEnvironment::CreateTexture(const char* Name, TEXT
     return pTexture;
 }
 
+void TestingEnvironment::SetDefaultCompiler(SHADER_COMPILER value)
```

Comment:
Why is this needed? Should we not use DEFAULT compiler everywhere except for the 6.0+ shader model?